src/app/App.js

```diff
@@ -15,34 +18,72 @@ class App extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      valueSeleted: null
+      listOdParticipantsAvailable: JSON.parse(localStorage.getItem('listOfParticipants')),
```

Comment:
a typo here `listOdParticipantsAvailable` should be `listOfParticipantsAvailable`